Permanent/Modules/EditMetadata/Views/MetadataEditView.swift

```diff
@@ -9,101 +9,107 @@ import SwiftUI
 
 struct MetadataEditView: View {
     @ObservedObject var viewModel: FilesMetadataViewModel
-    @State private var showAlert = false
     
     init(viewModel: FilesMetadataViewModel) {
         self.viewModel = viewModel
     }
-
+    
     var body: some View {
         CustomNavigationView {
-            VStack {
-                Group {
-                    SectionHeaderView(viewModel: viewModel)
-                    Divider()
-                        .padding(.horizontal, 0)
-                        .padding(.top, 24)
-                }
-                Group {
-                    VStack {
+            ZStack {
+                Color.whiteGray
+                    .frame(maxWidth: .infinity, maxHeight: .infinity)
+                    .edgesIgnoringSafeArea(.all)
+                ScrollView(showsIndicators: false) {
+                    LazyVStack {
+                        Group {
+                            SectionHeaderView(viewModel: viewModel)
+                            Divider()
+                                .padding(.horizontal, 0)
+                                .padding(.top, 24)
+                        }
+                        Group {
+                            VStack {
+                                SectionView(
+                                    assetName: "metadataDescription",
+                                    title: "Description",
+                                    rightButtonView: RightButtonView(
+                                        text: "Enter Description",
+                                        action: { print("Description tapped") }
+                                    ),
+                                    haveRightSection: false
+                                )
+                                
+                                ZStack {
+                                    RoundedRectangle(cornerRadius: 2)
+                                        .fill(Color.white)
+                                        .overlay(
+                                            RoundedRectangle(cornerRadius: 2)
+                                                .inset(by: 0.5)
+                                                .stroke(Color.galleryGray, lineWidth: 1)
+                                        )
+                                    TextView(text: $viewModel.inputText, didSaved: $viewModel.didSaved, viewModel: viewModel, textStyle: TextFontStyle.style39, textColor: .middleGray)
```

Comment:
This line is too long, can you format so that every parameter is one under the other?